forge-gui/res/cardsfolder/upcoming/vein_ripper.txt

```diff
@@ -0,0 +1,11 @@
+Name:Vein Ripper
+ManaCost:3 B B B
+Types:Creature Vampire Assassin
+PT:6/5
+K:Flying
+K:Ward:Sac<1/Creature/creature>
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Creature | TriggerZones$ Battlefield | Execute$ TrigLoseLife | TriggerDescription$ Whenever a creature dies, target opponent loses 2 life and you gain 2 life.
+SVar:TrigLoseLife:DB$ LoseLife | ValidTgts$ Opponent | TgtPrompt$ Choose an opponent to lose life | LifeAmount$ 2 | SubAbility$ DBGainLife
```

Comment:
I think just remove this `TgtPrompt` - we are targeting anyway so the default will be better